Projects/UOContent/Engines/Plants/PlantBowl.cs

```diff
@@ -1,190 +1,155 @@
+                return;
+            }
 
-            protected override void OnTarget(Mobile from, object targeted)
+            if (targeted is FertileDirt dirt)
             {
-                if (m_PlantBowl.Deleted)
-                {
-                    return;
-                }
+                var _dirtNeeded = Core.ML ? 20 : 40;
 
-                if (!m_PlantBowl.IsChildOf(from.Backpack))
+                if (!dirt.IsChildOf(from.Backpack))
                 {
                     from.SendLocalizedMessage(1042664); // You must have the object in your backpack to use it.
-                    return;
                 }
-
-                if (targeted is FertileDirt dirt)
+                else if (dirt.Amount < _dirtNeeded)
                 {
-                    var _dirtNeeded = Core.ML ? 20 : 40;
-
-                    if (!dirt.IsChildOf(from.Backpack))
-                    {
-                        from.SendLocalizedMessage(1042664); // You must have the object in your backpack to use it.
-                    }
-                    else if (dirt.Amount < _dirtNeeded)
-                    {
-                        // You need more dirt to fill a plant bowl!
-                        from.LocalOverheadMessage(MessageType.Regular, 0x3B2, 1061896);
-                    }
-                    else
-                    {
-                        var fullBowl = new PlantItem(true);
-
-                        if (from.PlaceInBackpack(fullBowl))
-                        {
-                            dirt.Consume(_dirtNeeded);
-                            m_PlantBowl.Delete();
-
-                            // You fill the bowl with fresh dirt.
-                            from.LocalOverheadMessage(MessageType.Regular, 0x3B2, 1061895);
-                        }
-                        else
-                        {
-                            fullBowl.Delete();
-
-                            // There is no room in your backpack for a bowl full of dirt!
-                            from.LocalOverheadMessage(MessageType.Regular, 0x3B2, 1061894);
-                        }
-                    }
+                    // You need more dirt to fill a plant bowl!
+                    from.LocalOverheadMessage(MessageType.Regular, 0x3B2, 1061896);
                 }
-                else if (IsDirtPatch(targeted))
+                else
                 {
-                    var fullBowl = new PlantItem();
+                    var fullBowl = new PlantItem(true);
 
                     if (from.PlaceInBackpack(fullBowl))
                     {
-                        m_PlantBowl.Delete();
+                        dirt.Consume(_dirtNeeded);
+                        _plantBowl.Delete();
 
-                        from.LocalOverheadMessage(MessageType.Regular, 0x3B2, 1061895); // You fill the bowl with fresh dirt.
+                        // You fill the bowl with fresh dirt.
```

Comment:
We had a discussion about this a while ago. I am ok with opening this discussion up. The reason the comments were moved is because they go over the line limit and the automatic style linter was chopping the arguments and then leaving the comment at the end on the same line as the close of the method.

Reply:
yeah I agree with the linter in general, but with these strings it does save time to quickly find the number :)